packages/flutter_tools/lib/src/commands/daemon.dart

```diff
@@ -6,6 +6,8 @@ import 'dart:async';
 import 'dart:convert';
 import 'dart:io';
 
+import 'package:usage/src/uuid.dart'; // ignore: implementation_imports
```

Comment:
I maintain `package:usage`, so while this is a bit of a shortcut I felt not much risk of APIs changing out from under us.

But I can inline the class - it is self-contained and small.